mods/gen4/formats-data.js

```diff
@@ -1588,7 +1588,7 @@ exports.BattleFormatsData = {
 		tier: "LC",
 	},
 	glalie: {
-		randomBattleMoves: ["spikes", "icebeam", "iceshard", "earthquake", "explosion", "taunt"],
```

Comment:
I think this one is less of an issue since STAB Ice Shard is at least useful for revenging the scary 4x weak Dragons. My reasoning for the change is that Glalie always wants Spikes, so the fewer other moves in its movepool the better, and I'd basically always rather have two of EQ/Explosion/Taunt instead of Ice Shard for the last two moves in any matchup without Salamence/Dragonite/Garchomp.